config/rbac/role.yaml

```diff
@@ -74,6 +82,48 @@ rules:
   - patch
   - update
   - watch
+- apiGroups:
```

Comment:
I think we're going to have to either diverge from kubebuilder and spilt this into multiple files, or we're going to have to be smarter with kustomize patches and configs so we can avoid requiring Kueue-specific CRDs in the MCAD only configs (and core MCAD-specific CRDs in the Kueue config).